commerce/domain/pricing/service/classes/PricingServiceSample.cls

```diff
@@ -141,6 +149,48 @@ public class PricingServiceSample extends commercestorepricing.PricingService {
     return txnResponse;
   }
 
+  private Map<String, PriceDataFromExternalService> getPriceFromStaticResponse(Set<String> productIds) {
+        String responseJson = '{';
+        for(String productID : productIds) {
+            Double price = 10.00;
+            Double listPrice = 10.00;
+            Double unitPrice = 10.00;
```

Comment:
Let's delete these since they aren't used.

Reply:
ok I was meaning to use listPrice for listPrice and unitPrice for unotPrice, let me do that